src/commands/configure.ts

```diff
@@ -0,0 +1,55 @@
+import cli from 'cli-ux';
+import { mkdirp, readJSON, writeFile } from 'fs-extra';
+import { join } from 'path';
+import BaseCommand, { FileConfig } from '../base';
+
+export default class Configure extends BaseCommand {
+  public static description = `Configure Vertex credentials.`;
+
+  public static examples = [
+    `$ vertex configure
+Saved 'https://platform.vertexvis.com' configuration to '~/.config/@vertexvis/vertex-cli/config.json'.
+`,
+  ];
+
+  public static flags = { ...BaseCommand.flags };
+
+  public async run(): Promise<void> {
+    const {
+      flags: { basePath },
+    } = this.parse(Configure);
+    const client = this.userConfig?.client;
+    const id = await cli.prompt(`Vertex client ID`, {
+      default: client?.id,
+      required: false,
+    });
+    const secret = await cli.prompt(`Vertex client secret`, {
+      default: client?.secret,
+      required: false,
+      type: 'hide',
+    });
+
+    const configPath = join(this.config.configDir, 'config.json');
```

Comment:
Thoughts on using a `.env` file? Then you wouldn't need to manage the json schema and the `.env` file could be used for other purposes as well.

https://github.com/motdotla/dotenv#readme

Reply:
Where would this file live on customer machines? Home directory? How would we distinguish between different environments?

Reply:
I don't know. Just trying to avoid implementing our own configuration management system.

Reply:
I'm using Oclif's approach to config, https://oclif.io/docs/config#docsNav